ggshield/core/git_hooks/prepush.py

```diff
@@ -0,0 +1,110 @@
+import logging
+import os
+import sys
+from typing import List, Optional, Tuple
+
+from ggshield.core.git_shell import git, is_valid_git_commit_ref
+from ggshield.core.text_utils import display_warning
+from ggshield.core.utils import EMPTY_SHA
+
+
+logger = logging.getLogger(__name__)
+
+
+OUTDATED_HOOK_MESSAGE = """The installed pre-push hook did not pass its command-line arguments to ggshield. This can cause the hook to fail if the name of the remote you are pushing to is not "origin".
```

Comment:
NIT: 🏗 There is a `ggshield/core/constants.py` file, should we use a similar file for this kind of constants within iac commands?

Reply:
IMO these are specific to prepush operations and would not benefit from being shared in a constants file. What do you think @Walz?

Reply:
I think If it's only used in this file, it can stay here.